Backend/makefile

```diff
@@ -0,0 +1,4 @@
+build:
+	docker build -t image .
```

Comment:
`image` is too generic IMHO. You could name it more verbosely like: `redisClient`.